app/jobs/publish_feed_job.rb

```diff
@@ -25,19 +26,14 @@ def perform(podcast, pub_item)
     PublishingPipelineState.settle_remaining!(podcast)
   end
 
-  def publish_feed(podcast, feed)
-    publish_apple(feed)
-    publish_rss(podcast, feed)
-  end
-
-  def publish_apple(feed)
+  def publish_apple(podcast, feed)
     return unless feed.publish_to_apple?
     res = PublishAppleJob.perform_now(feed.apple_config)
```

Comment:
Took me awhile to realize - this class is a job, but we _only_ call it synchronously.  So we block right here in this thread until Apple publishing works/fails.